quinn-h3/src/server.rs

```diff
@@ -130,12 +304,71 @@ impl Stream for IncomingConnection {
     }
 }
 
+/// HTTP/3 handshake future.
+///
+/// Represents an ongoing HTTP/3 handshake. Upon success, this future will resolve to a
+/// stream of [`IncomingRequest`]s.
```

Comment:
```suggestion
/// `Stream` of [`IncomingRequest`]s.
```

(Is it actually a `Stream`?)

Reply:
It implements `Stream`.